site/docs/main/api-reference.html

```diff
@@ -1,33 +1,27 @@
 <p>Packages:</p>
 <ul>
 <li>
-<a href="#projectcontour.io%2fv1">projectcontour.io/v1</a>
+<a href="#projectcontour.io%2fv1alpha1">projectcontour.io/v1alpha1</a>
```

Comment:
I don't know why this file was changed. Maybe another PR didn't update properly?

Reply:
Hmm, maybe an unstable sort within the doc generator? Looks like it switched the order of v1 and v1alpha1

Reply:
When I check out your branch and re-run the generate, it undoes this change..

Reply:
I think I spotted the issue in the API docs generator, TLDR it's using a map somewhere along the line which potentially throws away the sort

Reply:
Maybe just remove this file from the PR since you didn't actually change anything in the API?